SourceCode/GPS/Forms/Pickers/FormRecordPicker.Designer.cs

```diff
@@ -190,11 +190,11 @@ private void InitializeComponent()
         private System.Windows.Forms.ListView lvLines;
         private System.Windows.Forms.ColumnHeader chName;
         private System.Windows.Forms.Button btnDeleteField;
-        private System.Windows.Forms.Label label1;
+        private System.Windows.Forms.Label labelDeleteRecord;
         private System.Windows.Forms.Button btnDeleteAB;
-        private System.Windows.Forms.Label label2;
-        private System.Windows.Forms.Button btnOpenExistingLv;
+        private System.Windows.Forms.Label labelCancel;
+        private System.Windows.Forms.Button labelButtonOpenExistingLv;
```

Comment:
Here this is a button, so the prefix should be `button`:
```suggestion
        private System.Windows.Forms.Button buttonOpenExistingLv;
```